mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabUiActivator.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Frank Becker and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Frank Becker - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.mylyn.gitlab.ui;
+
+import java.io.File;
+
+import org.eclipse.mylyn.internal.tasks.core.ITasksCoreConstants;
+import org.eclipse.mylyn.internal.tasks.ui.TasksUiPlugin;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.osgi.framework.BundleContext;
+
+/**
+ * The activator class controls the plug-in life cycle
+ */
+public class GitlabUiActivator extends AbstractUIPlugin {
```

Comment:
do we need this class?

Reply:
After the change in GitlabImages we need the class because we now have the method getImageDescriptor with usage of imageDescriptorFromPlugin

Reply:
but if we have UI Activator we can use it in a standard way

1. override `initializeImageRegistry`
2. access images via `GitlabUiActivator.getDefault().getImageRegistry()`

Reply:
fixed with next commit